File: src/main/java/no/nav/veilarboppfolging/controller/response/KvpDTO.java
package no.nav.veilarboppfolging.controller.response;




import java.time.ZonedDateTime;


public class KvpDTO implements Comparable<KvpDTO> {
    private long kvpId;
    private long serial;
    private String aktorId;
    private String enhet;
    private String opprettetAv;
    private ZonedDateTime opprettetDato;
    private String opprettetBegrunnelse;
    private String avsluttetAv;
    private ZonedDateTime avsluttetDato;
    private String avsluttetBegrunnelse;

    @Override
    public int compareTo(KvpDTO k) {
        return Long.compare(serial, k.serial);
    }
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/KvpDTO.kt
package no.nav.veilarboppfolging.controller.response

import java.time.ZonedDateTime
import kotlin.Comparable
import kotlin.Int
import kotlin.String

data class KvpDTO(
    val kvpId: Long,
    val serial: Long,
    val aktorId: String,
    val enhet: String,
    val opprettetAv: String?,
    val opprettetDato: ZonedDateTime,
    val opprettetBegrunnelse: String?,
    val avsluttetAv: String?,
    val avsluttetDato: ZonedDateTime?,
    val avsluttetBegrunnelse: String?,

): Comparable<KvpDTO?> {
    override fun compareTo(other: KvpDTO?): Int {
        return (serial - (other?.serial ?: - 1)).toInt()
    }
}